addons/point_of_sale/static/src/js/models.js

```diff
@@ -1493,7 +1493,47 @@ exports.PosModel = Backbone.Model.extend({
             'total_included': sign * round_pr(total_included, this.currency.rounding),
         }
     },
+    get_taxes_after_fp: function(taxes_ids){
+        var self = this;
+        var taxes =  this.taxes;
+        var product_taxes = [];
+        _(taxes_ids).each(function(el){
+            var tax = _.detect(taxes, function(t){
+                return t.id === el;
+            });
+            product_taxes.push.apply(product_taxes, self._map_tax_fiscal_position(tax, self.order));
+        });
+        product_taxes = _.uniq(product_taxes, function(tax) { return tax.id; });
+        return product_taxes;
+    },
+    _map_tax_fiscal_position: function(tax, order = false) {
+        var self = this;
+        var current_order = this.get_order();
```

Comment:
We should consider the given `order` here.

Reply:
done